ReactApps/reacttest/src/card.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from "react";
+
+class ItemCard extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            title: "Название",
+            oldPrice: "цена",
+            newPrice: "цена"
+        };
+    }
+
+    componentWillMount(){
+        const {title, id} = this.props;
+        console.log("im card im rendered "+ `${title}: ${id}`)
+    }
+
+    render(){
+        return(
+            <div className="card">
+                <div className="header">
+                    <div className="icons">
+                        <button id="list"></button>
+                        <button id="like"></button>
+                        <button id="compare"></button>
+                    </div>
+                    <div className="photo">
+                    </div>
+                </div>
+                <div className="sectionBody">
+                    <div className="text">
+                        <div>
+                            <p className="itemTitle">{this.props.title}</p>
```

Comment:
You can destructure props {title, oldPrice, newPrice} = this.props before return and use it.